uint8 snack_id
uint8 count